clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -32,21 +36,30 @@
  * 
  * Metadata is maintained for only a subset of topics, which can be added to over time. When we request metadata for a
  * topic we don't have any metadata for it will trigger a metadata update.
+ * <p>
+ * If topic expiry is enabled for the metadata, any topic that is unknown or has not been used within the expiry
+ * period is removed from the metadata refresh set after an update. Consumers disable topic expiry since they explicitly
+ * manage topics while producers rely on topic expiry to limit the refresh set.
  */
 public final class Metadata {
 
     private static final Logger log = LoggerFactory.getLogger(Metadata.class);
 
+    public static final long TOPIC_EXPIRY_MILLIS = 5 * 60 * 1000;
```

Comment:
Shouldn't this be `TOPIC_EXPIRY_MS` to be consistent with how we typically name variables?